src/main/java/com/synopsys/integration/alert/channel/jira/util/JiraIssueFormatHelper.java

```diff
@@ -59,47 +66,89 @@ public String createTitle(final String provider, final LinkableItem topic, final
         // FIXME make this provider-agnostic
         String prettyPrintedKey;
         if (!componentKeys.getCategory().contains("Vuln")) {
-            prettyPrintedKey = componentKeys.prettyPrint(true);
+            prettyPrintedKey = componentKeys.prettyPrint(false);
```

Comment:
We would add all the vulnerabilities which caused the title to be greater than 255 characters.  Is there any information other than project, project version, component, component version, and category that we need in the title of the ticket?

Reply:
Otherwise I can collapse this if else.

Reply:
Policy Name

Also, your current code will set it to `false` every time. Maybe it is a problem with the collector? Do the vulnerability `LinkableItems` for policy tickets get flagged as `partOfKey`?